tests/unit_tests/data_model/test_metadata_collector.py

```diff
@@ -240,25 +234,102 @@ def test_collect_schema_dict(args_dict_site, tmp_test_directory):
     )
 
 
+def test_process_metadata_from_file():
+    _collector = metadata_collector.MetadataCollector({})
+
+    meta_dict_1 = {"cta": {"product": {"description": "This is a sample description"}}}
+    _dict_test_1 = _collector._process_metadata_from_file(meta_dict_1)
+    assert _dict_test_1 == meta_dict_1
+
+    meta_dict_2 = {"cta": {"PRODUCT": {"description": "This is a sample description"}}}
+    _dict_test_1 = _collector._process_metadata_from_file(meta_dict_2)
+
+    meta_dict_3 = {"cta": {"PRODUCT": {"description": "This is a sample\n description"}}}
+    _dict_test_1 = _collector._process_metadata_from_file(meta_dict_3)
```

Comment:
hmm, it seems copilot is repeating tests? I don't think we need to repeat the test with different inputs

Reply:
There is a small difference (and not from copilot): the second statement does include a `\n` and we want to test that it is removed.

Reply:
Ok!